build.sbt

```diff
@@ -16,7 +16,7 @@ libraryDependencies ++= Seq(
 
 //sbt-spark-package
 spName := "saurfang/spark-sas7bdat"
-sparkVersion := "2.2.0"
+sparkVersion := "2.4.2"
```

Comment:
Why are you bumping this? Does this PR need anything from Spark 2.4?

Reply:
Spark changed the signature of createInputStreamWithCloseResource in spark 2.4.

I think that can be re-worked to not use that method and keep 2.2 compat.